oci/layout/oci_transport_test.go

```diff
@@ -192,31 +199,37 @@ func TestReferencePolicyConfigurationIdentity(t *testing.T) {
 func TestReferencePolicyConfigurationNamespaces(t *testing.T) {
 	ref, tmpDir := refToTempOCI(t)
 	defer os.RemoveAll(tmpDir)
+
+	fullyExplicitPath, err := explicitfilepath.ResolvePathToFullyExplicit(tmpDir)
+	if err != nil {
+		assert.Fail(t, fmt.Sprintf("Unexected error: %v", err))
+	}
+
 	// We don't really know enough to make a full equality test here.
 	ns := ref.PolicyConfigurationNamespaces()
 	require.NotNil(t, ns)
 	assert.True(t, len(ns) >= 2)
-	assert.Equal(t, tmpDir, ns[0])
-	assert.Equal(t, filepath.Dir(tmpDir), ns[1])
-
-	// Test with a known path which should exist. Test just one non-canonical
-	// path, the various other cases are tested in explicitfilepath.ResolvePathToFullyExplicit.
-	//
-	// It would be nice to test a deeper hierarchy, but it is not obvious what
-	// deeper path is always available in the various distros, AND is not likely
-	// to contains a symbolic link.
-	for _, path := range []string{"/etc/skel", "/etc/skel/./."} {
-		_, err := os.Lstat(path)
-		require.NoError(t, err)
-		ref, err := NewReference(path, "sometag")
-		require.NoError(t, err)
-		ns := ref.PolicyConfigurationNamespaces()
-		require.NotNil(t, ns)
-		assert.Equal(t, []string{"/etc/skel", "/etc"}, ns)
-	}
+	assert.Equal(t, fullyExplicitPath, ns[0])
+	assert.Equal(t, filepath.Dir(fullyExplicitPath), ns[1])
+
+	//// Test with a known path which should exist. Test just one non-canonical
```

Comment:
This test does not work on macOS. I am not sure whether there is a directory which could be used across platform to make this work.

Reply:
Actually, after the `/var/tmp` vs. `/private/var/tmp` ambiguity is resolved somehow, this test can just as well use `ioutil.TempDir(…)` instead of relying on a pre-existing directory in the OS.

Reply:
The `/etc/skel` problem should have been …resolved… by #336.